keyboards/handwired/onekey/proton_c/config.h

```diff
@@ -29,4 +29,4 @@
 
 #define RGB_DI_PIN A0
 
-#define ADC_PIN A0
+#define ADC_PIN A1
```

Comment:
```suggestion
#define ADC_PIN A0
```

Reply:
Doesn't this make it share the pin with rgblight, @zvecr?

Reply:
I think the idea with the onekey boards is that although both are using the same pin, under normal circumstances only one of these is enabled at a time, depending on the keymap built.

Reply:
I made the change to A0, no worries.